### PR TITLE
 WFLY-15169 Refactor EJB client marshallers into a separate module

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.redhat.com/browse/WFLY-15169

### CODE REVIEW

**Comment by pferraro** [comment]
@rhusar @rachmatowicz Please review.

**Review by rhusar — APPROVED** [review]
Looks good to me; @rachmatowicz might want to have a look too.

File: clustering/ejb/client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline clustering/ejb/client/pom.xml:4]: 2010? I can't believe it but my wristwatch says it's 2021 already!

File: ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/clustering/ejb/client/main/module.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/clustering/ejb/client/main/module.xml:5]: BTW Same here.

File: clustering/ejb/client/pom.xml (hunk 1 above)
Comment (pferraro) [reply clustering/ejb/client/pom.xml:4]: :1st_place_medal:

**Review by rachmatowicz — APPROVED** [review]
Looks good.

**Comment by pferraro** [comment]
Added missing license for new module.

**Pull request merged**